lib/xclarity_client/configuration.rb

```diff
@@ -29,8 +33,13 @@ def self.default
         :username   => ENV['LXCA_USERNAME'],
         :password   => ENV['LXCA_PASSWORD'],
         :host       => ENV['LXCA_HOST'],
+        :port       => ENV['LXCA_PORT'],
         :verify_ssl => ENV['LXCA_VERIFY_SSL'] != "NONE"
       })
     end
+
+    def is_integer?(param)
+      param.to_i.to_s == param.to_s
```

Comment:
This don't smell goog. I recommend to use match method with a regex to do this.

should be something like that: `/^(\s*|\d+)$/.match(param) != nil`